app.js

```diff
@@ -0,0 +1,106 @@
+var Issue = can.DefineMap.extend({
+  seal: false
+}, {
+  id: 'number',
+  title: 'string',
+  sort_position: 'number',
+  body: 'string'
+});
+
+Issue.List = can.DefineList.extend({
+  '#': Issue
+});
+
+Issue.algebra = new can.set.Algebra(
+  can.set.props.id('id'),
+  can.set.props.sort('sort')
+);
+
+Issue.connection = can.connect.superMap({
+  url: '/api/github/repos/canjs/github-issues-example/issues',
+  Map: Issue,
+  List: Issue.List,
+  name: 'issue',
+  algebra: Issue.algebra
+});
+
+var socket = io();
+socket.on('issue created', function(issue) {
+  Issue.connection.createInstance(issue);
+});
+socket.on('issue updated', function(issue) {
+  Issue.connection.updateInstance(issue);
+});
+socket.on('issue removed', function(issue) {
+  Issue.connection.destroyInstance(issue);
+});
+
+can.view.callbacks.attr('sortable', function(element) {
+  $(element).sortable({
+    containment: 'parent',
+    handle: '.grab-handle',
+    revert: true,
+    start: function(event, ui) {
+      var draggedElement = ui.item;
+      draggedElement.addClass('drag-background');
+    },
+    stop: function(event, ui) {
+      var draggedElement = ui.item;
+      draggedElement.removeClass('drag-background');
+    },
+    update: function(event, ui) {
+      var draggedElement = ui.item[0];
+      var draggedIssue = (can.data.get.call(draggedElement, 'issue'))();
```

Comment:
I’m having a hard time finding the docs for the `data` helper. Can you point me in the right direction?